en/rocky/8/guides/rocky-8-installation.md

```diff
@@ -68,159 +67,190 @@ Insert and boot off the installation medium (optical disk, USB flash drive, and
+
+The System section of the _Installation Summary_ screen is used for customizing and making changes to the underlying hardware of the target system. This is where you create your hard drive partitions or volumes, specify the file system to be used, and specify the network configuration.
 
 ### Installation Destination
 
-From the Installation Summary screen, click the Installation Destination option. This takes you to the corresponding task area.
-You will see a screen displaying all the candidate disk drives that you have available on the target system. If you have only one disk drive on the system, as on our sample system, you will see the drive listed under Local Standard Disks with a check mark beside it. Clicking the disk icon will toggle on or off the disk selection check mark. We want it selected/checked here.
+From the _Installation Summary_ screen, click the <kbd>Installation Destination</kbd> option. This takes you to the corresponding task area.
 
+You will see a screen displaying all the candidate disk drives that you have available on the target system. If you have only one disk drive on the system, as on our sample system, you will see the drive listed under _Local Standard Disks_ with a check mark beside it. Clicking the disk icon will toggle on or off the disk selection check mark. We want it selected/checked here.
 
-Under the Storage Configuration Options section, select the Automatic radio button.
-Then click Done at the top of the screen.
-Once the installer determines that you have a usable disk, you will be returned to the installation summary screen.
+Under the _Storage Configuration_ Options section, select the <kbd>Automatic</kbd> radio button.
 
+Then click <kbd>Done</kbd> at the top of the screen.
 
+Once the installer determines that you have a usable disk, you will be returned to the _Installation Summary_ screen.
 
 ### Network & Hostname
+
 The final task of the installation procedure deals with network configuration, where you can configure or tweak network-related settings for the system.
 
-**NOTE:** After you click on the Network & Hostname option, all correctly detected network interface hardware (such as Ethernet, wireless network cards, and so on) will be listed in the left pane of the network configuration screen.Depending on the Linux distribution and the specific hardware setup, Ethernet devices in Linux have names similar to eth0, eth1, ens3, ens4, em1, em2, p1p1, enp0s3, and so on.
+> **NOTE:** After you click on the <kbd>Network & Hostname</kbd> option, all correctly detected network interface hardware (such as Ethernet, wireless network cards, and so on) will be listed in the left pane of the network configuration screen. Depending on the Linux distribution and the specific hardware setup, Ethernet devices in Linux have names similar to `eth0`, `eth1`, `ens3`, `ens4`, `em1`, `em2`, `p1p1`, `enp0s3`, and so on.
  
```

Comment:
I personally like the indented notes, but again, the kbd tags don't really seem to do much other than act as another way of adding code in either of my markdown editors...I'm going to request that @wsoyinka review your changes rather than continuing on myself. I think that would be best.

Reply:
Ok two reviews are always better than only one